rubin_sim/moving_objects/make_lsst_obs.py

```diff
@@ -209,34 +216,69 @@ def make_lsst_obs():
         logging.basicConfig(level=logging.INFO)
 
     # Read orbits.
-    orbits = mo.read_orbits(args.orbit_file)
+    orbits = mo.Orbits()
+    orbits.read_orbits(args.orbit_file)
 
+    if args.positions_file is not None:
+        position_data = np.load(args.positions_file)
+        object_positions = position_data["positions"].copy()
```

Comment:
For my own education - why does this need to be a copy?

Reply:
The `.copy()` loads it into memory, otherwise it'll read it off disk every time it gets referenced. And maybe it can't be accessed after doing a `.close()` if there's no `.copy()`